src/modules/icons.ts

```diff
@@ -13,5 +13,6 @@ export const icons = {
ddU6PlF+9ZosSwBjixRaPoC5wDTHNoNMi2vUV81q9TrLfRBuJqhaFZRwm1OgR2BS8lOlq860/Na7KhWlXphQvfhaDDWAGPL+tWb7OSk88heVa/CDcOq/FrPowQnBZWwHweVUOobFPpwPGA4FHdSotNlNvIw9RhgPNnRaGXoteb8Wsbn1+bDTaUeh9Dzi52kW4PxLDCwQsHqAxxLtkf7L4A3EhijN/Kf9fL8Z++jIpV6zEIfBqzWrcF4DTiw4kEbSbZvyV35g8Q3A4zDm/m/fVf+s4wEBqlCpZ6U0PMLvV6vBmUlcJaRe1dP9gZ2z8V6Vv7c4cb8G+XjwG+BWWQvby0AXgUW5p9X8//2Uv5nfpv/P5Pyv2Ns/neOzP+N3lZcqZdJ6H0D3RXJu5kE9DJ2Ikq91ULPL/I/9GkUzAX2NnYiSr0tQm8AfqlPo2AVcCG+VSj1F1tjlR7Ml0TqYzf1AgcD/9Sn0fAECW/uJdELbXtgaj7duodSL5nQ8wv8rB6Nin8Cp6kfqbHIPs+7t9FW6mUUen6BP9Cj0fEwnmIjbZfXNnkvrQ+lXlKhdyF76UXiYhFwOtComqQVc+XnAIvfp8eUetmEnl/cQLK1vBIfTwHD1ZRsYpZ3p2W7Yir1sgl9rUbwuLo4WQ1cDWymsmQD+e0JfIfWvQmu1Msm9PziRgDL9We0LADGOA0ja2W2fT41t7CNvaXUyyb0/OI+TvZ6usTLdGCEOqu8zA8GnqthXyn1sgl9Lal7px4/k4FdVFvl5PMh4KE69ZRSL5vQ84s71Dn1JGgi22/9A6qu9MLZCfgR9d9rXqmXTehr3Qn8VWcmI/Y78Di0MkpmKHArxW59rdTLJvT84gYA/6Mvk1oRcwcudSyDWD4M3JP/sg6BUi+b0POL6wTcpCuTYg3wKHAEbvyVkkga8unOR/IxDI1SL5vQ17rA43Ev9RR5HvgcJT/XNHF5dAfOBH4fYf8o9TIKPb/AgcDPdWSSLCI78HiYCo1qfvxq4K3Ie0epl1Hoa13kaOB1HZnsdMyvgVOAbmq1cEH0AE4DnoxkWkWpxyv1sUVeZO/87mKVjkyWJflD1COA9uq2blnpABwG3AYsTbhflHpZhb7WhQ7JpdCkH5PmlXxK5iDlXpNctCfbTuOmkn2brbLUt6PYjQzHhrzYDwAT8C3TMvBGfjf5SaC7et7kDGwGjCJbN/5aifujylLfrcBnHmNjuOD+wCWkfdq2/It3yJZAXgjsgZuDrd3rjWQ7lZ4PPE619kGqstRHVUbo6zT7oflduw9Qy8NCslfQzwCGV0nweU/vRnaAxH35N5kqU2Wp31wpoa/nodABwBXAbyn2NWapL4vzu9NxwCco0cnyZG9KfxK4CpjC+58CpNSrI/TeBdyojk2lGN3zB0YXkm0uNRM3AysTi4BfAdeSHVK8H9Av4n7sl/+Mp+c/8xS/VSr1TeibMxX6xgu0BdmmYCOAE8gOdDgTOBc4z0/yn8/nYzuoiKkaslfpewM7A4cAJ+fz3d8B7id7i3apPlbqreyvzmSrw+rF131SJVUJ0125lCfnD22nkJ3BOgt4OV+J0KxnlXqd+/A/61jPi026VCVIb+lPpR5BH25P/d74vcCkS1WCtEB3KvVIenFanep4hkmXqoRolt5U6pH04lV1quHxJl2qIvTHdaZSj6QXj61T/Q4x6VIVod+pL5V6JL24Q51q55GSUhmhX64rlXokvdie2u862wx0MulSFaH/u55U6hH1Y623gZhvyqVKQt9TRyr1iPpxXo1r9TNTLlUSehc8XEWpx9OPf61xna405VI1qc/Uj0o9kl5cVOMajTLhUjWh36gblXoEfdiV2m8z0d+ES9WEfpJeVOoR9OEHa1yX2aZbqij0frgBl1IP34efrXFNrjfdUlWpP6MTlXrgHpxQ43oca7KlqkK/VB8q9YD91wDMrWEdlgLdTLZUVehDdaFSD9h/+9a4Bnebaqm61H+rC5V6oN77fo2vf6SJlqoL/Yt6UKkH6Lv+wPIaXvcrQAcTLVUXei88G1SpF99342t8zb4dKpKH62YdqNQL7LddqO3WE6uAQSZZJAvYTrgmXakX02sdgP+p8XX+yBSLvDto9+k/pV5An11Z4+tbA3zIBIu8O2i7eZeu1OvcY6fmAq4lPza9IusP3A91X6mlvmfA3hpJ7bdsXg3sYnJF1h+67YC3dV9pWQIcHqCvxuTyrTU3mFqRjYfvMr1XalYC5wMNBfRSJ+A7dZhmgWwP9S1MrMjGQ9gVmK33Ss/Pge3r2Ee7A8/W8ef/vGkV2bQwHuQD0krwDjAO6FvD3hlEdnhKUx1/7l8W8Q1DpExSv1bfVYZlwPXAPq0RJdAeOAy4k/qfVbu4nt8sRMoq9C7A87qucswHbgU+B3wU6Lu25IFGsn1Y9gPOBO4BFhb4851iOkVaJ/VhuM+LZNNvi/NeWBPw57jVVIq0TerHBw6xCPkD1i4mUqTtUv+GPpGA/B3Y1iSK1EboDfgWqYRhCbC7KRSprdQ7kq1dFimK5cAI0ydSH6l3BaboGSmAlcAnTJ1IfaXeHXhU30id78w/btpEipF6Z+AnekfqNGd+qCkTKVbq7an9uZDiapYPmy6RcGI/K5/vFGkL04CBJkokvNT3AxboJGklt+BLQyJRSX1L4Ke6SVrAW8DJpkckXrF/gWzfD5GN8Qi+/SmShNQHAPfrLFkPrwOjTYlIemI/Apilw4Rsn/Srgd4mQyRdqbcHTgPm6rRK0gTcAexoGkTKI/aOudi9Y68Gy4EJwAfsfpHyir2B7Liy+6j/cWVSPHOAS4D+drtIteTeHzgXeAoP0Uj9QecE4FCg0c4WUe4Dgc+T7RHzho6MmtXA74ArgAOADnawiGxsWmZnYEy+MuJx4K9kZ1tKsSwFZgJ3AhcCI4DudqmItFX0nYAhZFsNHAuckt/VnwOc56fVn3OBM/NfoCfk0v4QsEXK/fL/AL08QclcNueFAAAAAElFTkSuQmCC'
```

Comment:
This base64 image is larger than I would like, but to keep everything internal to the extension this is the easiest path for now.